build/xtask/src/flash.rs

```diff
@@ -164,7 +164,7 @@ pub fn config(
         | "nucleo-h753zi" | "stm32h7b3i-dk" | "gemini-bu-1" | "gimletlet-1"
         | "gimletlet-2" | "gimlet-a" | "gimlet-b" | "psc-1" | "sidecar-1"
         | "stm32g031" | "stm32g070" | "stm32g0b1" | "hifive-inventor"
-        | "hifive1-revb" => {
+        | "hifive1-revb" | "opentitan" => {
```

Comment:
Is the chip named opentitan or is it opentitan earlgrey? (https://docs.opentitan.org/hw/top_earlgrey/doc/)

Reply:
Hm, it looks like it is "OpenTitan Earl Grey".  I would say that we should use the proper chip name, but also most the time it has just been referred to as the opentitan in our own docs.  Qemu also calls it the opentitan; however, the qemu naming conventions for the whole system is a bit suspect (it incorrectly calls all of the opentitan peripherals `ibex`).

Looking at the other chip names for baseline, I think Ill just change it to `opentitan_earlgrey`, unless someone has objections.

Reply:
From [tock documentation](https://github.com/tock/tock/blob/master/boards/README.md#out-of-tree-boards), it looks like `opentitan` is the project name and `earlgrey` is the chip/MCU name.

`opentitan_earlgrey` here sounds good to me.